api/envoy/service/ratelimit/v3/rls.proto

```diff
@@ -114,6 +121,9 @@ message RateLimitResponse {
       // Point in time at which the quota expires.
       google.protobuf.Timestamp valid_until = 2;
     }
+
+    // The unique id that is associated with each Quota either at individual descriptor level or whole descriptor set level.
```

Comment:
Can you comment on what the implementation is expected to do with this? I assume for logging, stats, etc.?

Reply:
This id can be used to share the same quota among multiple descriptors or descriptor sets. @tyxia can you add this clarification, please?